contracts/Registry/BasicRegistry.sol

```diff
@@ -47,4 +46,9 @@ contract BasicRegistry is IRegistry {
     function exists(bytes32 id) public view returns (bool) {
         return items[id] != 0x0;
     }
+
+    function _remove(bytes32 id) internal {
```

Comment:
why did you break this out?

Reply:
moved this to internal so it can be called from the extending contracts directly. It's used in `resolveChallenge()` now to remove the item and bypass all the required checks from the public `remove()` function.